website/content/docs.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Docs"
```

Comment:
I'd like to just call this "Book" to avoid confusion with rustdoc

Reply:
Should we rename Rustdoc to Docs then?
I kinda don't like the name rustdoc

Reply:
How about calling rustdocs "API" like what https://rocket.rs/ does.

Reply:
Should we call "book" "guide" as well?
Seems to be clearer to me then

Reply:
sure